### DISCUSSION TITLE
EddieHub Fest

### DISCUSSION THREAD

**Discussion by vinzvinci** [EddieHubCommunity/support#1928] [Ideas]
We will be having a fun community event every year called **EddieHub Fest**. This event will run for 1 week.

We can do:
- Community members can host a workshop `(3-5 workshops every day is enough)`
- Guest speaker
- Game activity `(it will be cool if we prepare more games)`
- Talent shows `(folks can do art, playing music instruments, singing and etc.)`
- Design competition?
- Rebranding `(create a new design for EddieHub community projects)`
- Fun call on discord `(we can talk random topics?)`
- Open Source Week 
- Rewarding day 
  - Best in collaboration
  - Best in contributing to open source
  - Etc. 
- Etc.

Please do suggest so we can have a fun event every year! I'm so excited to hear more ideas!

**Comment by verma-kunal** [comment]
This would be an amazing experience and i would definitely love to be a part of this & also help in organising such events !!
Full support to the idea ✨

**Reply by vinzvinci** [reply]
Great!. Yeah, we need more organizers/hosts for this event. Not just Eddie, but all legitimate hubbers xD. This event will be awesome!

**Reply by Akshu-on-github** [reply]
I think it would be much easier once the Ambassador role is set up - alternatively, we could use this to select Community Ambassadors too 🤔

Really cool idea @vinzvinci!

**Reply by vinzvinci** [reply]
Hmm, let's see. I think Eddie wants to combine the community anniversary with this event.

**Comment by ItsMeIshan** [comment]
This event will be awesome 🥳 and it will be my first time in EddieHub fest!! We can have a session of ***talent hunt*** if anyone wants to present something like art, dubbing, acting , doing a rap etc. This would increase number of collaborations🎉

**Reply by vinzvinci** [reply]
Good idea, it can! it's up to them if they want to reveal their secret talent xD. I updated the discussion and added talent shows.

**Reply by Akshu-on-github** [reply]
I've heard that people have been using discord stages for this, maybe we could look into it as an option?
Or Twitter Spaces for that matter

**Reply by vinzvinci** [reply]
I recommend discord xD

**Comment by ghost** [comment]
:100: But Maybe I'll Call It HubberFest? And can it better be something like mlh init, like a hackathon oriented fest?

**Reply by Akshu-on-github** [reply]
Maybe we could conduct an MLH member event over the weekend?

It could be ~36 hours to kick off the event and the results would be announced at the end of the week-long fest (if it's at the end, there might be people bugging those organising the event for results)

Alternatively, we could make our own rules and host our Hackathon independently for the entire week - but imho, that would be a lot on our plate

**Reply by vinzvinci** [reply]
Good idea @Akshu-on-github & @krishdevdb 👍

**Reply by ashwinexe** [reply]
> Maybe we could conduct an MLH member event over the weekend?
> 
> It could be ~36 hours to kick off the event and the results would be announced at the end of the week-long fest (if it's at the end, there might be people bugging those organising the event for results)
> 
> Alternatively, we could make our own rules and host our Hackathon independently for the entire week - but imho, that would be a lot on our plate

This is a great idea, that way we can invite more folks to the event.

**Reply by WebShivam** [reply]
yeah this is really great idea , we should work on it, if everybody permits on it

**Comment by Vyvy-vi** [comment]
Interesting, I think Eddie was planning on hosting something like this, next year or this year. (not sure when), but this would be really amazing! (We've been having many discussions about this, since like 27th Jan, 2021 and Nick's been suggesting platform ideas, etc. for a while) 😉  Something is coming up for y'all pretty soon (and maybe we can collect ideas from here to get an idea of what the community would like, and work on that)

> [nhcarrigan]
I wonder how people would feel about a show and tell day.
[Eddie]
Oh that could be fun, I was chatting with Sara about EddieCon today too. Maybe one of each per year or something

<del>(this is a spoiler. Eddie please don't be angry 😂 )</del>
> [Eddie]
I had some more ideas for the EddieCon, I will add it to the webpage and show it to you soon - the idea was a "wellness break" - in the middle of the conf, I was thinking that maybe we have a break to do 15mins of yoga, and I will do it too, so people can laugh and how bad I am at it. Hopefully break up the sitting down for so long

Oh wait this isn't a spoiler... Eddie posted it in `#ideas`

Also, Eddie already has a discussion open for this - https://github.com/EddieHubCommunity/support/discussions/789

**Reply by Vyvy-vi** [reply]
`EddieCon Beta v0.1` can be a mini-event coming soon :D

**Comment by eddiejaoude** [comment]
So many great ideas from everyone, we love it 💥 

I think the main ideas that have been discussed recently are: 
- 20k live stream celebration with workshop and give aways etc
- EddieCon
- Ambassadors program
- EddieFest (btw this needs a new name, it sounds a bit weird to me 😂 )

Some of my initial thoughts:

| Event  | Effort | Lead time |
| :--- | :--- | :--- |
| 20k celebration | Low | 2 weeks |
| EddieCon | Medium | 2 months |
| Ambassadors program | High | not sure |
| EddieFest | High | not sure |

We should definitely capture these ideas and their sub ideas, so we can start planning. I would like to get sponsors for some of these events so we can do more give aways.

Thank you all for making the suggestions and going into details on the ideas for each type of event. Keep the ideas coming 👍 

I am currently planning the 20k celebration live stream, where we will have a workshop for everyone to follow a long and give aways for the winners (more ideas welcome)

**Reply by vinzvinci** [reply]
Yeah good idea, let's combine the ideas together 👍

**Reply by ghost** [reply]
btw I already gave a name idea: https://github.com/EddieHubCommunity/support/discussions/1928#discussioncomment-962395

**Comment by SahityaRoy** [comment]
> We will be having a fun community event every year called **EddieHub Fest**. This event will run for 1 week.
> 
> We can do:
> 
> * Community members can host a workshop `(3-5 workshops every day is enough)`
> * Guest speaker
> * Game activity `(it will be cool if we prepare more games)`
> * Talent shows `(folks can do art, playing music instruments, singing and etc.)`
> * Design competition?
> * Rebranding `(create a new design for EddieHub community projects)`
> * Fun call on discord `(we can talk random topics?)`
> * Open Source Week
> * Rewarding day
>   
>   * Best in collaboration
>   * Best in contributing to open source
>   * Etc.
> * Etc.
> 
> Please do suggest so we can have a fun event every year! I'm so excited to hear more ideas!

This is a cool event i am happy to be a part of this event

**Comment by Panquesito7** [comment]
What about a coding event? Probably like making a calculator and review their codes. 🙂

**Reply by Akshu-on-github** [reply]
Something similar to this? https://github.com/EddieHubCommunity/support/discussions/1928#discussioncomment-962395

**Reply by Panquesito7** [reply]
> Something similar to this? [#1928 (comment)](https://github.com/EddieHubCommunity/support/discussions/1928#discussioncomment-962395)

Yeah, something similar to that. 🙂
